src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,72 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = TimespanParameterSet, HelpUri = "")]
```

Comment:
Hi, since this cmdlet does not have a HelpUri, please add code to skip it in the Online help tests located at 
https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1